JSDemos/Demos/Common/ActionAndListsOverview/jQuery/index.html

```diff
@@ -45,7 +45,7 @@
         </div>
         <div class="tile">
           <script type="text/html" id="tile-template">
-            <img class="tile-image" src="../../../../images/hotels/{{ FileName }}" />
+            <img alt="{{FileName}}" class="tile-image" src="https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/hotels/{{ FileName }}" />
```

Comment:
i dont get it) why absolute path

Reply:
Replaced: [d223abbadd3d41efe5957c9016e5e242fc8c67e1].